faqs/zh/cdp.md

```diff
@@ -139,9 +139,9 @@ CDP是无需许可的，任何人都可以创建和使用。没有繁琐的申
 
 CDP 系统运行在以太坊区块链上，所有的智能合约和交易都是公开透明的。任何人都可以通过访问 [MakerDAO 统计面板][6]，或直接查询区块链亲自审计系统的偿付能力。
 
-[1]:	https://github.com/makerdao/community/blob/master/faqs/cdp.md#what-is-the-collateralization-ratio
-[2]:	https://www.investopedia.com/personal-finance/apr-apy-bank-hopes-cant-tell-difference/
-[3]:	https://github.com/makerdao/community/blob/master/faqs/liquidation.md#what-is-the-liquidation-ratio
-[4]:	https://www.investopedia.com/terms/l/leverage.asp
-[5]:	https://github.com/makerdao/community/blob/master/faqs/stability-fee.md#what-is-the-stability-fee
+[1]:	./cdp.md#%E4%BB%80%E4%B9%88%E6%98%AF%E6%8A%B5%E6%8A%BC%E6%AF%94%E7%8E%87
```

Comment:
Are we going to prefer URL encoded links? Is there a problem with using the unicode?

Reply:
I think the Chinese sub-title got encoded automatically in the editor. It does not affect the display but will try to change.

Reply:
Cool, thanks @Chaology